scripts/create_changelog.py

```diff
@@ -0,0 +1,329 @@
+        answer = python_utils.INPUT().lower()
+        if answer in ['y', 'ye', 'yes']:
+            break
+
+    current_release_version = branch_name[len(
+        common.RELEASE_BRANCH_NAME_PREFIX):]
+    source_branch = 'develop'
+    target_branch = 'update-changelog-for-releasev%s' % current_release_version
+    release_summary_lines = []
+    # This complete code block is wrapped in try except since in case of
+    # an exception, we should ensure that the changes made to AUTHORS,
+    # CONTRIBUTORS, CHANGELOG and about-page are reverted and the branch
+    # created is deleted. This is required to avoid errors when running
+    # the script again.
+    try:
+        with python_utils.open_file(
+            feconf.RELEASE_SUMMARY_FILEPATH, 'r') as release_summary_file:
+            release_summary_lines = release_summary_file.readlines()
+
+        update_changelog(
+            release_summary_lines, current_release_version)
+        update_authors(release_summary_lines)
+        update_contributors(release_summary_lines)
+        update_developer_names(release_summary_lines)
+
+        python_utils.PRINT(
+            'Creating new branch with updates to AUTHORS, CONTRIBUTORS, '
+            'CHANGELOG and about-page...')
+        sb = repo_fork.get_branch(source_branch)
+        repo_fork.create_git_ref(
+            ref='refs/heads/%s' % target_branch, sha=sb.commit.sha)
+
+        for filepath in [
+                ABOUT_PAGE_FILEPATH, CONTRIBUTORS_FILEPATH, AUTHORS_FILEPATH,
+                CHANGELOG_FILEPATH]:
+            contents = repo_fork.get_liness(filepath, ref=target_branch)
+            with python_utils.open_file(filepath, 'r') as f:
+                repo_fork.update_file(
+                    contents.path, 'Update %s' % filepath, f.read(),
+                    contents.sha, branch=target_branch)
+        common.run_cmd(GIT_CMD_TEMPLATE_CHECKOUT % ('%s %s %s %s') % (
+            ABOUT_PAGE_FILEPATH, CONTRIBUTORS_FILEPATH, AUTHORS_FILEPATH,
+            CHANGELOG_FILEPATH))
+        common.open_new_tab_in_browser_if_possible(
+            'https://github.com/oppia/oppia/compare/develop...%s:%s?'
+            'expand=1' % (github_username, target_branch))
+        python_utils.PRINT(
+            'Pushed changes to Github. '
+            'Please create a pull request from the %s branch' % target_branch)
+    except Exception as e:
+        common.run_cmd(GIT_CMD_TEMPLATE_CHECKOUT % ('%s %s %s %s') % (
+            ABOUT_PAGE_FILEPATH, CONTRIBUTORS_FILEPATH, AUTHORS_FILEPATH,
+            CHANGELOG_FILEPATH))
+        # The get_git_ref code is wrapped in try except block since the
+        # function raises an exception if the target branch is not found.
+        try:
+            repo_fork.get_git_ref('heads/%s' % target_branch).delete()
+        except Exception:
+            python_utils.PRINT(
+                'Please ensure that %s branch is deleted before re-running '
```

Comment:
Maybe have a check at the start of the script to ensure that the branch to be created isn't yet present. (Better than relying on the developer to read each message carefully -- helps mitigate human error.)

Reply:
Done.